crates/ruff_linter/src/rules/ruff/rules/sequence_sorting.rs

```diff
@@ -922,7 +922,16 @@ fn multiline_string_sequence_postlude<'a>(
     // ```
     let newline_chars = ['\r', '\n'];
     if !postlude.starts_with(newline_chars) {
-        return Cow::Borrowed(postlude);
+        // Special-case a few common situations so we can get formatting for the closing
+        // parenthesis that is similar to what the formatter would produce
+        return match postlude {
+            ")" | "]" | "}" => Cow::Owned(format!("{newline}{leading_indent}{postlude}")),
+            ",)" | ",]" | ",}" => {
+                let closing_paren = postlude.chars().last().unwrap();
+                Cow::Owned(format!(",{newline}{leading_indent}{closing_paren}"))
+            }
+            _ => Cow::Borrowed(postlude),
+        };
```

Comment:
If we wanted to _always_ add a trailing comma to the last item here, we could do something like this instead:

```suggestion
        let parenthesis_re = regex::Regex::new(r"^,?([\])}])$").unwrap();
        if let Some(captures) = parenthesis_re.captures(postlude) {
            if let Some(match_) = captures.get(1) {
                let closing_paren = match_.as_str();
                return Cow::Owned(format!(",{newline}{leading_indent}{closing_paren}"));
            }
        }
        return Cow::Borrowed(postlude);
```